package.json

```diff
@@ -16,17 +16,25 @@
     "@babel/core": "^7.20.2",
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^12.1.2",
     "babel-loader": "^9.1.0",
     "eslint": "^8.28.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.11",
+    "global-jsdom": "^8.6.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.3.0",
+    "jest-environment-jsdom": "^29.3.1",
+    "jsdom": "^20.0.2",
     "prettier": "2.8.0"
   },
   "scripts": {
     "build": "webpack",
     "lint": "eslint .",
     "lint:fix": "eslint --fix",
-    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
+    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
+    "test": "jest"
   }
```

Comment:
Can we add a separate `jest` key here with `root` field to find which directories to look up for the test files, so when the `jest` command executed it only go to jest key in the `package.json` and look for the only directories which needs testing instead of looking everywhere? Is this a good practice to add @Ron-Lavi ?

Reply:
Imho for our project, it's enough to use the default `rootDir` which checks for .test.js files

Reply:
Okay got it.